### PR TITLE
Akupara take two

### PR DESCRIPTION
**Pull request by tonotom1**
## About the PR
This PR adds the Akupara: a full-feature botanical vessel meant to be immediately capable of performing mutations, and interact with the resulting possibilities that occur from said mutations without having to tide a chemistry-capable vessel or construct too much additional equipment. It looks like a turtle by sheer coincidence!

## Why / Balance
I love botany and everything about it. My current method of accessing everything botany forces me to buy a chemistry-capable ship and then cramming it into the poor Garden, or setting up trays in a ship that has no or little botanical identity at all. Though the Garden is good at its job it has very little space for interpersonal interaction. I tried creating something that was significantly more roomy, and in an attempt to differentiate it from the Garden, gave it more capabilities but not too robustly where it could be abused stock.

## How to test
Buy it and inspect it, or load it using mapping & loadgrid

## Media
image

TORTL!
image

## Breaking changes
It broke before from guidebook changes like I suspected, and it'll probably break again. However I did the fixing and it's quite easy for me to fix.

**Changelog**
-add: Akupara, a medium full-feature botany ship to the Frontier shipyard console intended to provide a platform to experience all aspects of botany possible out the gate, but not necessarily be the best compared to a custom solution.

### CODE REVIEW

**Comment by MagnusCrowe** [comment]
image
You might want to re-orient the mass scanner to be facing the same direction as the shuttle console.  See how the radars are rotated 90 degrees from each other?
image
Perhaps something like this?

image
Something is wrong with the piping here...

image
I recommend moving the gas condenser into the atmos room and moving the grinder into the chem lab. 

image
The use of windoor here seems bad to me.  On end of it just kinda floats, know what I mean?  Also the gravity generator is inaccessible.
image
The substation is also inaccessible, can you figure out a way to re-organize this room so that all the machines are accessible? 

image
This nook looks good.
image
Is there a reason that this vending machine isn't in this empty space?

**Comment by tonotom1** [comment]
Great catch on the fucked storage loop! No idea how I caused that to happen. The nutrimax vendor is there due to the presence of the suit wall locker and my desire to maintain the "park bench" composition going directly above. Since I am disallowed from mapping a soft drink vendor I had to settle for utilizing the nutrimax as a stand-in.

File: RobustToolbox
Comment (tonotom1) [inline RobustToolbox:1]: HOW DO I UNDO THIS

I AM SO SICK OF FILES RANDOMLY DECIDING TO CHANGE WHEN I HAVEN'T DONE A THING TO THEM  WHY ARE THERE 137 FILES CHANGED

Comment (tonotom1) [inline RobustToolbox:1]: this file has sabotaged my branch again and i must delete everything for the umpteenth time

Comment (dvir001) [reply RobustToolbox:1]: We can fix it for you, just keep working on the same PR this isn't an issue till it's ready anyway.

Comment (tonotom1) [reply RobustToolbox:1]: thankyou i am sort of on wits end having to fight github every turn of the corner

**Comment by Cheackraze** [comment]
love the idea and the concept, my only concern is the massive amount of windows there are in one section. These pieces here in particular just kind of look off to me:
image
I believe we do have diagonal glass window pieces, you could try using some of those and breaking up the long stretches of glass a bit with some solid wall pieces to break it up a tad more so its not a pure glass window section thats super prone to lasers and emp bolts

**Comment by tonotom1** [comment]
uh oh i don't think i merged it correctly just there

**Comment by tonotom1** [comment]
> love the idea and the concept, my only concern is the massive amount of windows there are in one section. These pieces here in particular just kind of look off to me:(image) I believe we do have diagonal glass window pieces, you could try using some of those and breaking up the long stretches of glass a bit with some solid wall pieces to break it up a tad more so its not a pure glass window section thats super prone to lasers and emp bolts

Here's what it looks like after another aesthetics pass

File: RobustToolbox
Comment (tonotom1) [reply RobustToolbox:1]: i fixed it myself!! yay!

**Comment by tonotom1** [comment]
Been a few days of braining over it and haven't felt the impulse to change anything further. Should be properly settled for another round of review now.

**Comment by MagnusCrowe** [comment]
image
One last nitpick.  No custom fill crates.  Also there's no hotplate, you should probably replace the custom crate with a table and hotplate.   If you want a couple jugs, replace the potted plant with a rack and put a few on top of the rack.

**Comment by MagnusCrowe** [comment]
image
Also recommend you re-arrange your power room like this.

**Comment by tonotom1** [comment]
Screenshot 2024-09-01 114437
Screenshot 2024-09-01 114959
Screenshot 2024-09-01 114413
feng shui was established

**Review by MagnusCrowe — APPROVED** [review]

**Pull request merged**